app/translators/transaction.translator.js

```diff
@@ -4,6 +4,12 @@ const BaseTranslator = require('./base.translator')
 const Joi = require('joi')
 
 class TransactionTranslator extends BaseTranslator {
+  constructor (data) {
+    super(data)
+
+    this.chargeFinancialYear = this._financialYear(this.chargePeriodStart)
```

Comment:
Just for reference. @StuAA78 has quite rightly pointed out that we don't get the finance year when we are creating a transaction. This is because of logic in the `CalculateChargeService` that changes the result depending on whether the request was to create a transaction or just to see the calculation results.

We've discussed some options to try and resolve it whilst avoiding duplicating the logic around generating and validating the finance year. Stuart is on the case! 🕵️